Platform Documentation.md

```diff
@@ -339,6 +339,8 @@ $ cctrlapp APP_NAME/DEP_NAME deploy
 To deploy a specific version, append your version control systems identifier (full commit-SHA1 for Git or an integer for Bazaar).
 If not specified, the version to be deployed defaults to the latest image available (the one built during the last successful push).
 
+Running [workers](#Workers) have to be redeployed separately by initiating a [manual restart](#restarting-workers).
```

Comment:
Also it seems the deploy command does have a --resstart-workers parameter. We should note this here too.

`$ cctrlapp idcdemo1/default deploy --restart-workers`